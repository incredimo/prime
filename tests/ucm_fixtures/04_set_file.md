```ucm set#s1 path="./out.txt" mode="write"
Hello
```