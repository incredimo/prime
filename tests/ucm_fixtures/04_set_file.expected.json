{
  "id": "s1",
  "verb": "set",
  "attrs": {
    "path": "./out.txt",
    "mode": "write"
  }
}